ClinGen/clincoded
# Connects to #1549. Connects to #1550. Segregation LOD Score.

**Steps to test:**
1. In a Gene-Disease Record, start a new **Family** curation.
2. Fill out the **required** fields.
3. In the **LOD Score** section in the **Family — Segregation** panel, select the **Yes** option from the **Published LOD score?** dropdown menu.
4. Next, enter **1.2** in the **Published Calculated LOD score** text field.
5. Then select the **Yes** option from the **Include LOD score in final aggregate calculation?** dropdown menu.
6. Now expect to see the **Sequencing Method** dropdown menu being shown with an **asterisk** marking it as **required**.
7. Select the **Candidate gene sequencing** option from the dropdown menu and save the form without providing a _Proband_.
![image](https://user-images.githubusercontent.com/6956310/44123895-448c21c2-9fdf-11e8-82c1-ace72ef13caa.png)
8. Start another new **Family** curation, and repeat the above steps, _but_ enter **3.45** in the **Published Calculated LOD score** text field and select the **Exome/genome or all genes sequenced in linkage region** option from the dropdown menu.
9. Upon returning to record's page, click on the **Classification Matrix** button at the top-right.
10. On the subsequent matrix page, expect to see the updated segregation data and UI being shown as below:
![image](https://user-images.githubusercontent.com/6956310/44124096-4d132614-9fe0-11e8-9a10-69719844d94c.png)
11. Click on the **Save** button below the matrix and proceed to the subsequent page. Expect to see the same segregation data being shown consistently in the matrix.
12. Click on the **Evidence Summary** button below the matrix and open up a new window/tab. Expect to see the same segregation data being shown consistently in the matrix.
13. Also expect to see new segregation columns being added to case-level and case-level segregation tables.
